__tests__/app.test.js

```diff
@@ -369,6 +369,51 @@ describe("api/articles/:article_id/comments ", () => {
  })
 })
 
+describe("api/comments/:comment_id ", () => {
+    test("DELETE: 204 with no content", () => {
```

Comment:
204 means no content. In this case we would not expect a body to be returned at all